docs/assets/joining_parallel_gateway_doc.bpmn

```diff
@@ -0,0 +1,233 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" xmlns:modeler="http://camunda.org/schema/modeler/1.0" id="Definitions_1a8cbo5" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="5.0.0" modeler:executionPlatform="Camunda Cloud" modeler:executionPlatformVersion="8.0.0">
+  <bpmn:process id="Process_0zaz4fy" isExecutable="true">
+    <bpmn:task id="Activity_0np5087" name="Increment number of taken sequence flows">
+      <bpmn:incoming>Flow_0sekarg</bpmn:incoming>
+      <bpmn:outgoing>Flow_1avosat</bpmn:outgoing>
+    </bpmn:task>
+    <bpmn:startEvent id="Event_1igkqew" name="Take outgoing sequence flows">
```

Comment:
You've named the events using verbs, which are generally better fitting for activities. Likewise, you've named the activities using the naming scheme of "an object and a verb reflecting a state", which is typically better fitting for events. This might confuse readers about what has happened and what needs to be done.

🔧 Please follow the [recommended naming guidelines](https://docs.camunda.io/docs/components/best-practices/modeling/naming-bpmn-elements/).